### DISCUSSION TITLE
Welcome to yaru Discussions!

### DISCUSSION THREAD

**Discussion by didrocks** [ubuntu/yaru#2676] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Feichtmeier** [comment]
Hello 👋
I'm Frederik and I mostly work on the gtk and the GNOME shell theme in the yaru project. From time to time I try to contribute also with icon designs 😅

**Reply by elsiehupp** [reply]
Ooh, icon design is a lot of fun!

**Comment by elsiehupp** [comment]
Hi! I'm Elsie. Most of my recent engagement has been with [Nextcloud Desktop](https://github.com/nextcloud/desktop/), mostly because the folks there are really nice. I'm a bit new to Linux development, so my focus is a bit scattershot right now, but in general the things I have been trying to engage with a "papercuts"; that is, things that annoy me personally, lol.

My programming experience is in Python, then Java, then C# (mostly in the context of [Unity 3D](https://unity.com/)), a tiny bit of C, then a bit of C++ (now mostly in the context of Nextcloud Desktop). I am a double grad school dropout. (Don't ask about my student loans... [sardonic cry-laugh-emoji])

I spend the most time using macOS (because reasons), but I prefer working in GNOME, mostly because (unlike Windows and macOS), you can often actually fix the things that annoy you. (Also closed-source developers are kind of creepy.) So in addition to (somewhat begrudgingly) working in C++/Qt, I'm hoping to learn Vala/GTK and refresh my knowledge of C.

> As an aside, I have basically zero experience with KDE, and I have literally zero experience with anything else other than Pantheon. My impression with KDE is that the design paradigms are a bit more like Windows than like macOS, though obviously the analogy is a bit loose, so my preference for GNOME is mostly a parallel for my preference for macOS over Windows. My preference for GNOME and KDE over the others is mainly due to the fact that they seem to be the biggest players, so contributing to them would have the biggest impact. I don't begrudge anyone who uses a more "boutique" desktop shell, though.

I've spent a fair bit of time using Ubuntu, but more recently I've been using Debian, mostly because it's vanilla GNOME. I started using Debian after nerfing my computer (thanks, LVM!), but in resurrecting it I set up partitions to allow me to distro-hop, so I'm planning on reinstalling Ubuntu, as well. My various DigitalOcean droplets are all still Ubuntu, though.

My Linux computer is an Alienware Alpha connected to a smallish 1080p television on the other side of my living room, which unfortunately makes working directly on it less pleasant than it could be. Because I'm not going to be buying a gigantic 1440p or 2160p monitor anytime soon, a disproportionate amount of my Linux development time is over SSH from my MacBook with its superior ergonomics (i.e. sprawling on the sofa with it on my lap). I could probably install a Linux VM on my MacBook, but dual-booting seems a bit beyond the complexity I'd like to deal with for the time being. idk, maybe I'll do it regardless, though.

I also have a really old Mac Mini that I got for the purpose of setting up Matrix bridge for my iMessage so I can do iMessage on Linux. Yay iMessage. [I'm just radiating enthusiasm.](https://www.theverge.com/2021/4/9/22375128)
